modules/cluster-logging-elasticsearch-retention.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-elasticsearch.adoc
+
+[id="cluster-logging-elasticsearch-retention_{context}"]
+= Configuring log retention time
+
+You can specify how long the default Elasticsearch log store keeps indices 
+using a separate _retention policy_ for each of the three log sources: 
+infrastructure logs, application logs, and audit logs. The retention policy, 
+configured using the `maxAge` parameter in the Cluster Logging Custom Resource 
+(CR), determines when to delete the Elasticsearch indices. 
+
+If you do not create a retention policy, logs are retained for seven days by
+default. If you do not specify a retention policy for all three log sources,
+logs from only the sources with a retention policy are stored. For example, 
+if you do not set a retention policy for audit logs, the audit logs will not
+be retained and there will be no *audit-* index in Elasticsearch or Kibana. 
+
+.Prerequisites
+
+* Cluster logging and Elasticsearch must be installed.
+
+.Procedure
+
+To configure the log retention time:
+
+. Edit the Cluster Logging CR to add or modify the `retentionPolicy` parameter:
++
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+...
+spec:
+  managementState: "Managed"
+  logStore:
+    type: "elasticsearch"
+    elasticsearch:
+      nodeCount: 3
+    retentionPolicy: <1>
+      application:
+        maxAge: 1d
+      infra:
+        maxAge: 7d
+      audit:
+        maxAge: 7d
+...
+----
+<1> Specify the time that Elasticsearch should retain each log source. Enter an 
+integer and a time designation: weeks(w), hours(h/H), minutes(m) and seconds(s). 
+For example, `1d` for one day. Logs older than the `maxAge` are deleted. 
+By default, logs are retained for seven days. 
+
+. You can verify the settings in the Elasticsearch Custom Resource (CR).
++
+For example, the following Elasticsearch CR has a retention policy to roll over 
```

Comment:
For example, the cluster logging operator will generate the following....

Need to make it clear this is a managed resource